src/entities/sentry/ui/sentry-exception/sentry-exception.vue

```diff
@@ -16,10 +16,10 @@ const exceptionFrames = computed(() => {
   const frames = props.exception.stacktrace.frames || [];
 
   if (props.maxFrames > 0) {
-    return frames.reverse().slice(0, props.maxFrames);
+    return frames.slice(0, props.maxFrames);
   }
 
-  return frames;
+  return [...frames].reverse();
```

Comment:
Question: why do we need to use copy of array? I think better to use (frames | []).reverse() to keep performance better.